cirq/study/sweepable.py

```diff
@@ -14,7 +14,7 @@
 
 """Defines which types are Sweepable."""
 
-from typing import cast, Iterable, List, Union
+from typing import cast, Dict, Iterable, List, Union
```

Comment:
Pylint didn't catch that `Dict` is not used?

Reply:
Done.